### ISSUE TITLE
Unable to change color of admonition (block) title

### ISSUE THREAD

**Issue by ctargett** [asciidoctor/asciidoctor-pdf#561]
When an admonition block has a title, the color of the title is green. I'd like to change to match our scheme. I see from https://github.com/asciidoctor/asciidoctor-pdf/blob/master/docs/theming-guide.adoc#block that we can set the font-color, but assume that will make the entire block that color, when I only want to modify the title.

I tried:

```
admonition:
  title:
     font-color: '#7a2518'
```

and 

```
block:
  title:
     font-color: '#7a2518'
```

Am I missing a syntax?

**Comment by mojavelinux** [comment]
It's not possible at the moment to style block titles (aka captions) individually. You can control all block titles using the top-level caption key:

```
caption:
  font_color: #7a2518
```

In the future, we'll probably need to add a subkey to each block type to control the caption individually.

**Comment by ctargett** [comment]
I was going to say that the caption isn't working, but then noticed that Atom has been putting extra spaces in my lines in the theme.yml file :-(

Fixed that and it's working now. I can live with the caption option until more options are available. Thanks!

**Comment by mojavelinux** [comment]
This was almost solved by #307, but we missed the admonition category. I'll go ahead and extend that change to support admonition blocks as well.

Keep in mind the caption for all admonition blocks will share the same styles. In other words, it wouldn't be possible to set different properties per admonition type.
